jsdocs/index.html

```diff
@@ -0,0 +1,129 @@
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+        </a>
+        <div class="logo">
+            
+            
+        </div>
+        <div class="menu">
+            
+            <div class="navigation">
+                <a
+                    href="index.html"
+                    class="link"
+                >
+                    Documentation
+                </a>
+                
+                
+                
+            </div>
+        </div>
+    </div>
+</div>
+    <div id="main">
+        <div
+            class="sidebar "
+            id="sidebarNav"
+        >
+            
+            <nav>
+                
+                    <h2><a href="index.html">Documentation</a></h2><div class="category"><h3>Global</h3><ul><li><a href="global.html#addIndicators">addIndicators</a></li><li><a href="global.html#addNewGameLine">addNewGameLine</a></li><li><a href="global.html#addTargetListener">addTargetListener</a></li><li><a href="global.html#applyParameters">applyParameters</a></li><li><a href="global.html#changePieceDisplay">changePieceDisplay</a></li><li><a href="global.html#createNewRound">createNewRound</a></li><li><a href="global.html#displayLoosePopup">displayLoosePopup</a></li><li><a href="global.html#displayWinPopup">displayWinPopup</a></li><li><a href="global.html#duplicate">duplicate</a></li><li><a href="global.html#generateCombination">generateCombination</a></li><li><a href="global.html#getAvailableColors">getAvailableColors</a></li><li><a href="global.html#getColorApparition">getColorApparition</a></li><li><a href="global.html#getCombinationPlacement">getCombinationPlacement</a></li><li><a href="global.html#getCurrentCombination">getCurrentCombination</a></li><li><a href="global.html#getGameDomElements">getGameDomElements</a></li><li><a href="global.html#hideUnwantedColor">hideUnwantedColor</a></li><li><a href="global.html#nbColors">nbColors</a></li><li><a href="global.html#nbPossibilities">nbPossibilities</a></li><li><a href="global.html#nbTurns">nbTurns</a></li><li><a href="global.html#removeColorFromTarget">removeColorFromTarget</a></li><li><a href="global.html#removeTargetListener">removeTargetListener</a></li><li><a href="global.html#setDragListenerOnPieces">setDragListenerOnPieces</a></li><li><a href="global.html#setPieceDragData">setPieceDragData</a></li><li><a href="global.html#setTargetBackgroundColor">setTargetBackgroundColor</a></li><li><a href="global.html#setTargetDropEffect">setTargetDropEffect</a></li><li><a href="global.html#startNewGame">startNewGame</a></li><li><a href="global.html#verifyCurrentCombination">verifyCurrentCombination</a></li></ul></div>
+                
+            </nav>
+        </div>
+        <div class="core" id="main-content-wrapper">
+            <div class="content">
+                <header class="page-title">
+                    <p></p>
+                    <h1>Home</h1>
+                </header>
+                
+
```

Comment:
strange but don't impact the code

Reply:
this is the code generated by the jsdoc plugin, not ours ahah